netty/src/main/java/io/grpc/netty/NettyServer.java

```diff
@@ -272,9 +272,7 @@ public void initChannel(Channel ch) {
           transportListener = listener.transportCreated(transport);
         }
 
-        /**
-         * Releases the event loop if the channel is "done", possibly due to the channel closing.
-         */
+        /* Releases the event loop if the channel is "done", possibly due to the channel closing. */
```

Comment:
What's with the two javadoc that we changed away from being javadoc?

Reply:
Errorprone raised https://errorprone.info/bugpattern/NotJavadoc. As I understand, because this is an anonymous class.

Reply:
Oh, I see. They are both local classes (not anonymous). That makes sense.